index.html

```diff
@@ -0,0 +1,54 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Fork Me! FCC: Test Suite Template</title>
```

Comment:
Please try to give your project a good title.

Reply:
- [OPTIONAL] Nice one @wassimchakib, but try as much as you can to include `WHY` the student is asked to make of a suggestion as this would make it clearer for the student why he or she is being asked to do a certain thing. And  if possible, you can also direct the student to a resource where where more information can got for  more clarification.